CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
+## [1.0.6] - 2024-09-08
+### Add RichText Extensions
+- Add more extensions methods in:
+  - `String` extensions
+      - `RichText` extensions
```

Comment:
nit. fix indentation

Reply:
sounds good. i will get the practice down soon.

Reply:
Next time if there are merge conflicts, instead of closing the PR we can talk about conflict resolution.